src/plugins/mozprefs/mozprefs.c

```diff
@@ -235,12 +235,36 @@ static void writeKey (FILE * fp, const Key * parentKey, const Key * key)
 	if (realPrefName) elektraFree (realPrefName);
 	if (argString) elektraFree (argString);
 }
-
+static bool elektraCheckForInvalidMetaKey (Key * parentKey, KeySet * keySet)
+{
+	Key * cur = 0;
+	for (elektraCursor it = 0; it < ksGetSize (keySet); ++it)
+	{
+		cur = ksAtCursor (keySet, it);
+		const KeySet * metaKeys = keyMeta (cur);
+		for (elektraCursor jt = 0; jt < ksGetSize (metaKeys); ++jt)
+		{
+			const Key * meta = ksAtCursor (metaKeys, jt);
+			const char * pos = (const char *) keyName (meta);
+			if (elektraStrNCmp (pos, "meta:/internal/mozprefs", 19) != 0 && elektraStrCmp (pos, "meta:/origname") && elektraStrNCmp (pos, "meta:/rename", 12) != 0 && elektraStrCmp (pos, "meta:/binary") != 0)
```

Comment:
Why did you only compare the first 19 characters in `elektraStrNCmp (pos, "meta:/internal/mozprefs", 19)`?

Please use a consistent coding style (`elektraStrCmp (pos, "meta:/origname")` was not explicitly compared against `0`).

```suggestion
			if (elektraStrNCmp (pos, "meta:/internal/mozprefs", 23) != 0 && elektraStrCmp (pos, "meta:/origname") != 0 && elektraStrNCmp (pos, "meta:/rename", 12) != 0 && elektraStrCmp (pos, "meta:/binary") != 0)
```